ext/civigrant/CRM/Grant/DAO/Grant.php

```diff
@@ -228,33 +228,33 @@ public static function &fields() {
         'grant_contact_id' => [
           'name' => 'contact_id',
           'type' => CRM_Utils_Type::T_INT,
-          'title' => ts('Contact ID'),
-          'description' => ts('Contact ID of contact record given grant belongs to.'),
+          'title' => E::ts('Contact ID'),
+          'description' => E::ts('Contact ID of contact record given grant belongs to.'),
           'required' => TRUE,
           'where' => 'civicrm_grant.contact_id',
           'export' => TRUE,
           'table_name' => 'civicrm_grant',
           'entity' => 'Grant',
-          'bao' => 'CRM_Grant_BAO_Grant',
+          'bao' => 'CRM_Grant_DAO_Grant',
           'localizable' => 0,
           'FKClassName' => 'CRM_Contact_DAO_Contact',
           'html' => [
             'type' => 'EntityRef',
-            'label' => ts("Contact"),
+            'label' => E::ts("Contact"),
           ],
           'add' => '1.8',
         ],
         'grant_application_received_date' => [
           'name' => 'application_received_date',
           'type' => CRM_Utils_Type::T_DATE,
-          'title' => ts('Application received date'),
-          'description' => ts('Date on which grant application was received by donor.'),
+          'title' => E::ts('Application received date'),
+          'description' => E::ts('Date on which grant application was received by donor.'),
           'import' => TRUE,
           'where' => 'civicrm_grant.application_received_date',
           'export' => TRUE,
           'table_name' => 'civicrm_grant',
           'entity' => 'Grant',
-          'bao' => 'CRM_Grant_BAO_Grant',
+          'bao' => 'CRM_Grant_DAO_Grant',
```

Comment:
@colemanw any reason why this is being changed from the BAO to DAO?

Reply:
:shrug: GenCode wanted to...